### PR TITLE
Refactor manager entrance and add latest animations

### PR DESCRIPTION
**Pull request by gre**

### CODE REVIEW

**Review by juan-cortes — APPROVED** [review]
🚀

File: src/renderer/components/ListenDevices.js [hunk 1]
```diff
@@ -8,20 +8,11 @@ const ListenDevices = () => {
   const dispatch = useDispatch();
   useEffect(() => {
     let sub;
-    let timeout;
     function syncDevices() {
       const devices = {};
       sub = command("listenDevices")().subscribe(
         ({ device, deviceModel, type }) => {
           if (device) {
-            clearTimeout(timeout);
```
Comment (valpinkman) [inline src/renderer/components/ListenDevices.js:17]: there was a comment on this section that has not been ported: 

`// when there are no more device after 5s debounce, we'll restart the listen`

right after  `if (device) {`

so not sure if we want to remove it or not

**Review by valpinkman — APPROVED** [review]

File: src/renderer/components/ListenDevices.js (hunk 1 above)
Comment (gre) [reply src/renderer/components/ListenDevices.js:17]: i don't see the comment here, but i'm trying to remove this workaround. we'll have to test if we have problem in future.

Comment (valpinkman) [reply src/renderer/components/ListenDevices.js:17]: 👍

**Pull request merged**